src/main/java/edu/ksu/canvas/attendance/controller/SummaryController.java

```diff
@@ -100,42 +109,49 @@ private ModelAndView studentSummary(String sectionId, String studentId, boolean
         List<AttendanceSummaryModel> summaryForSections = isSimpleAttendance ?
                 reportService.getSimpleAttendanceSummaryReport(validatedSectionId) :
                 reportService.getAviationAttendanceSummaryReport(validatedSectionId);
-        List<LtiLaunchData.InstitutionRole> institutionRoles = canvasService.getRoles();
 
         student.getAttendances().sort(Comparator.comparing(Attendance::getDateOfClass).reversed());
 
-        summaryForSections.stream()
-                .flatMap(summary -> summary.getEntries().stream())
-                .filter(entry -> entry.getStudentId() == validatedStudentId)
-                .findFirst()
-                .ifPresent(entry ->  addAssignmentSummaryToPage(page, entry, isSimpleAttendance, student, assignment));
-        institutionRoles.stream()
-                .filter(institutionRole -> institutionRole.equals(LtiLaunchData.InstitutionRole.Learner))
-                .findFirst()
-                .ifPresent(role -> page.addObject("isStudent", true));
+        List<AttendanceSummaryModel.Entry> entries = new ArrayList<>();
 
-        page.addObject("sectionId", sectionId);
-        page.addObject("student", student);
-        page.addObject("summaryForm", makeupForm);
-        return page;
-    }
+        if (student.getSisUserId() != null) {
+            entries = summaryForSections.stream().flatMap(model -> model.getEntries().stream()).filter(entry -> student.getSisUserId().equals(entry.getSisUserId())).collect(Collectors.toList());
```

Comment:
This line is pretty long. Some versions of intellij will autoformat streams for you. If you are on an old version you can add a new line for every call after stream(). 

Sometimes that is excessive so I usually just use my best judgement to make it readable.